tests/functional/test_install.py

```diff
@@ -543,7 +546,7 @@ def test_install_argparse_shadowed(script):
 
 
 @pytest.mark.network
-@pytest.mark.skipif("sys.version_info < (3,4)")
+@pytest.mark.skipif("pip._vendor.six.PY2")
```

Comment:
I would also make a `python3_only` version and use it here. We may also move them somewhere in `tests.lib` so they can be used in other places, too.